From 1ac21753cf11ca025a3b73641cba3ca5acff28fe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 22 Apr 2020 14:12:47 +0200
Subject: [PATCH 08/12] Blacklist test_pickletools from __all__ test as it
 imports numpy

The test imports numpy, bringing its warnings along with it and causing
the test to fail.
---
 Lib/test/test___all__.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Lib/test/test___all__.py b/Lib/test/test___all__.py
index 6368ef0225..6583755e9b 100644
--- a/Lib/test/test___all__.py
+++ b/Lib/test/test___all__.py
@@ -80,6 +80,8 @@ def test_all(self):
         denylist = set([
             # Will raise a SyntaxError when compiling the exec statement
             '__future__',
+            # imports numpy which causes warnings
+            'test.test_pickletools',
         ])
 
         if not sys.platform.startswith('java'):
-- 
2.35.1

